src/xdocs/config_design.xml

```diff
@@ -277,6 +277,26 @@ public class FooTest {
         <source>
 &lt;module name=&quot;FinalClass&quot;/&gt;
         </source>
+        <p>
+          Examples :
+          
+class Parent{
```

Comment:
Code should be surrounded by `<source>` tag.

Reply:
done